public/assets/js/angular-controller.js

```diff
@@ -159,122 +159,27 @@ omsApp.controller('sidebarController', function($scope, $rootScope, $state) {
 });
 
 
-
```

Comment:
The example code from the template, that you see when you go to the template preview.